excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImpl.java

```diff
@@ -150,9 +154,19 @@ public DefinitionFileUploadMetadata importFormDefinitions(InputStream inputStrea
         logger.debug("Importing spreadsheet: Case types...");
 
         final CaseTypeParser caseTypeParser = parserFactory.createCaseTypeParser(parseContext);
-        final ParseResult<CaseTypeEntity> parsedCaseTypes = caseTypeParser.parseAll(definitionSheets);
+        Set<String> missingUserRoles = Sets.newHashSet();
```

Comment:
it shouldn't be ImportServiceImpl's concern to initialise `missingUserRoles`

Reply:
we can move this to importServiceImpl but the problem is the current error loggin hapenning in caseTypeService.createAll(jurisdiction, caseTypes); will not be executed..
if we are fine to ignore those checking and logging errors then we can move that to importServiceImpl